packages/loopring_v3/contracts/amm/AmplifiedAmmController.sol

```diff
@@ -55,8 +55,17 @@ contract AmplifiedAmmController is IAmmController, Claimable
     {
         address pool = msg.sender;
 
+        if (balances.length != vBalancesNew.length) {
+            return false;
+        }
+
         // Check if a curve change was explicitly authorized
         if (consumeCurveChangeAuthorized(pool)) {
+            for (uint i = 0; i < balances.length; i++) {
+                if (vBalancesNew[i] < balances[i]) {
+                    return false;
+                }
+            }
```

Comment:
Any reason to disallow AF < 1?

Reply:
In theory we can, but shall we do that?

Reply:
With or without the check it kind of depends on us. If we don't want to do it we just don't set any AF < 1.